examples/function/Cargo.toml

```diff
@@ -11,6 +11,8 @@ crate-type = ["cdylib"]
 
 [dependencies]
```

Comment:
I would rather put this into separate example (example: custom-logger) instead of overloading existing example.  I want to keep `function` simplest possible